examples/app.jsx

```diff
@@ -189,6 +190,14 @@ var squareStyles = {
     ':hover': {
       background: 'red',
     }
+  },
+  three: {
+    ':hover': {
+      background: 'yellow'
+    }
+    , ':disabled': {
```

Comment:
I think this file might not go through eslint, but we have standard comma style instead of comma first.